cfgov/legacy/housing_counselor/geocoder.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import absolute_import, unicode_literals
+
+from six import print_
+
+
+def geocode_counselors(counselors, **kwargs):
+    """Fill in missing latitude/longitude data for housing counselors."""
+    return ZipCodeBasedGeocoder(**kwargs).geocode(counselors)
+
+
+class ZipCodeBasedGeocoder(object):
+    """Fill in missing latitude/longitude data using zipcode locations.
```

Comment:
This is *basically* what we do now, but I think it's inadequate, especially since we display results on a map